### PR TITLE
chore(ci): get minikube tests working for external contributors

### PR DESCRIPTION
**Pull request by edvald**
**What this PR does / why we need it**:

We were authenticating with gcloud for just one specific integ test in the minikube tests, and that was breaking test execution for external contributors.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -85,17 +85,22 @@ commands:
           paths: [dashboard/node_modules]
           key: dashboard-v4-{{ checksum "dashboard/package-lock.json" }}
 
-  configure_kubectl_context:
+  install_kubectl:
     description: Configure the kubectl context so that we can access our remote cluster. Used for e2e testing.
```
Comment (eysi09) [inline .circleci/config.yml]: Can you update the description.

File: garden-service/test/integ/src/plugins/kubernetes/container/build.ts [hunk 2]
```diff
@@ -55,7 +82,7 @@ describe("k8sBuildContainer", () => {
       await init("local")
     })
 
-    it("should build a simple container", async () => {
+    it("should build a simple container (local only)", async () => {
```
Comment (eysi09) [inline garden-service/test/integ/src/plugins/kubernetes/container/build.ts]: Is there a better to specify whether a test is local/remote than just grepping the description?

Comment (edvald) [reply garden-service/test/integ/src/plugins/kubernetes/container/build.ts]: Not that I could find, no.

File: .circleci/config.yml (hunk 1 above)
Comment (edvald) [reply .circleci/config.yml]: Done

**Pull request merged**